### PR TITLE
Self-Targeting Docs

### PR DESCRIPTION
**Pull request by katemacfarlane**
Adding docs with instructions on how to setup your DevCycle Identity and use Self-Targeting for a feature

### CODE REVIEW

File: docs/essentials/targeting.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ sidebar_position: 8
 
 Targeting rules can be used to grant features to specific user groups, incrementally roll out features for monitoring, or create and test different feature configurations by using [multiple variables](/essentials/variables).
 
+:::tip Already understand the targetting essentials?
```
Comment (ajwootto) [inline docs/essentials/targeting.md:8]:
```suggestion
:::tip Already understand the targeting essentials?
```

File: docs/extras/advanced-targeting/self-targeting.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Self-Targeting
+sidebar_position: 2
+---
+
+Self-Targeting is a new DevCycle QA & debugging feature that allows you to easily serve yourself different variations of a feature without having to modify targeting rules in each environment. To leverage this feature, you must set up your DevCycle Identity by associating an SDK User ID to your DevCycle user profile. 
+
+![Self-Targeting Slideout on Feature Page](/aug-2023-self-targeting-slideout-open-feature-form.png)
+
+On each feature, you will see a Self-Targeting section within the Users & Targeting tab. This section will show you at-a-glance if there you have any active self-targeting overrides for feature, and how many other team-members within the current project have self-targeting overrides set as well.
```
Comment (ajwootto) [inline docs/extras/advanced-targeting/self-targeting.md:10]:
```suggestion
On each feature, you will see a Self-Targeting section within the Users & Targeting tab. This section will show you at-a-glance if you have any active self-targeting overrides for feature, and how many other team-members within the current project have self-targeting overrides set as well. 
```

Comment (ajwootto) [inline docs/extras/advanced-targeting/self-targeting.md:10]:
```suggestion
On each feature, you will see a Self-Targeting section within the Users & Targeting tab. This section will show you at-a-glance if there you have any active self-targeting overrides for a feature, and how many other team-members within the current project have self-targeting overrides set as well. 
```

Comment (ajwootto) [inline docs/extras/advanced-targeting/self-targeting.md:10]:
```suggestion
On each feature, you will see a Self-Targeting section within the Users & Targeting tab. This section will show you at-a-glance if there you have any active self-targeting overrides for feature, and how many other team members within the current project have self-targeting overrides set as well. 
```

File: docs/extras/advanced-targeting/self-targeting.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+sidebar_position: 2
+---
+
+Self-Targeting is a new DevCycle QA & debugging feature that allows you to easily serve yourself different variations of a feature without having to modify targeting rules in each environment. To leverage this feature, you must set up your DevCycle Identity by associating an SDK User ID to your DevCycle user profile. 
+
+![Self-Targeting Slideout on Feature Page](/aug-2023-self-targeting-slideout-open-feature-form.png)
+
+On each feature, you will see a Self-Targeting section within the Users & Targeting tab. This section will show you at-a-glance if there you have any active self-targeting overrides for feature, and how many other team-members within the current project have self-targeting overrides set as well. 
+
+Within the slide-out, you can setup your DevCycle Identity, select Self-Targeting overrides, and see the other Team Members that are currently self-targeting for that feature, including the variation and environment they have active overrides in.
```
Comment (ajwootto) [inline docs/extras/advanced-targeting/self-targeting.md:12]:
```suggestion
Within the slide-out, you can setup your DevCycle Identity, select Self-Targeting overrides, and see the other team members that are currently self-targeting for that feature, including the variation and environment they have active overrides in. 
```

File: docs/extras/advanced-targeting/self-targeting.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+![Self-Targeting Slideout on Feature Page](/aug-2023-self-targeting-slideout-open-feature-form.png)
+
+On each feature, you will see a Self-Targeting section within the Users & Targeting tab. This section will show you at-a-glance if there you have any active self-targeting overrides for feature, and how many other team-members within the current project have self-targeting overrides set as well. 
+
+Within the slide-out, you can setup your DevCycle Identity, select Self-Targeting overrides, and see the other Team Members that are currently self-targeting for that feature, including the variation and environment they have active overrides in. 
+
+## DevCycle Identity
+
+Setup your DevCycle Identity by entering the user ID you are providing to the DevCycle SDK, and which uniquely identifies you. Doing so unlocks debugging features such as Self-Targeting, a feature that allows you to override targeting rules to deliver different variations of the feature to yourself.
```
Comment (ajwootto) [inline docs/extras/advanced-targeting/self-targeting.md:16]: should there be an example of what this means exactly? Like maybe a code snippet showing the userId being passed to the SDK, and more of an explanation of what we mean by "your user ID"?

Comment (katemacfarlane) [reply docs/extras/advanced-targeting/self-targeting.md:16]: @ajwootto that'd be great! who can help me with that?

Comment (katemacfarlane) [reply docs/extras/advanced-targeting/self-targeting.md:16]: @ajwootto i'll add this later but get the docs out there today!

**Pull request merged**